cypress/support/fragments/exportManager/exportManagerSearchPane.js

```diff
@@ -95,7 +96,11 @@ export default {
         .find(MultiColumnListCell(integrationName))
         .click(),
     );
-    cy.wait(4000);
+    // cy.wait(4000);
```

Comment:
please delete this line if we don't need it

Reply:
Done